mail_template_restrict_by_default/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1 @@
+- Holger Brunn \<mail@hunki-enterprises.com> (https://hunki-enterprises.com)
```

Comment:
@mart-e @StefanRijnhart thank you now already for your input. any complaints about (names ordered alphabetically by surname)

```suggestion
- Holger Brunn \<mail@hunki-enterprises.com> (https://hunki-enterprises.com)
- Stefan Rijnhart \<stefan@opener.amsterdam> (https://opener.amsterdam)
- Martin Trigaux \<no.mail@github.profile> (https://odoo.com)
```

Reply:
@hbrunn thanks for the credit, you can set my `mat` odoo email address

Reply:
Not sure I deserve any credits but otherwise this is fine.